bigchaindb/backend/localmongodb/schema.py

```diff
@@ -32,118 +64,22 @@ def create_database(conn, dbname):
 @register_schema(LocalMongoDBConnection)
 def create_tables(conn, dbname):
     for table_name in backend.schema.TABLES:
-        logger.info('Create `%s` table.', table_name)
         # create the table
         # TODO: read and write concerns can be declared here
-        conn.conn[dbname].create_collection(table_name)
+        try:
+            logger.info(f'Create `{table_name}` table.')
+            conn.conn[dbname].create_collection(table_name)
+            create_indexes(conn, dbname, table_name, INDEXES[table_name])
```

Comment:
What if we just add new index?

Reply:
We can. But we would need to change logging then.  Unconditional `Creating indexes for ..` messages are disorienting.

Reply:
Then we should change logging and ensure all the indexes exist too

Reply:
Why?

Reply:
I was thinking about it. Most of our indexes come with a unique constraint. Unique constraints are not backwards-compatible and thus can not be introduced by a new `bigchaindb start` or `bigchaindb init`.

On the other hand, we are losing a good means of adding those rare indexes without constraints.. So, I think it's better to arrange it how you suggest.